warhorn/source/utils.py

```diff
@@ -617,6 +602,22 @@ def install_depen(dependency, dependency_name, logfile, print_log_name,
                         logfile, print_log_name)
             setDone(1)
 
+def package_version(package, filepath):
+    """
+    For each package finds the details in the filepath.
+
+    :return:
+    version_package : Package and its version details.
+    filepath : The filepath from where version_package is derived.
+    """
+    version_package = False
+    if not os.path.exists(filepath):
+        filepath = "../requirements.txt"
+    with open(filepath, 'r') as requirements:
+        for line in requirements:
+            if package in line:
+                version_package = line
+    return version_package, os.path.abspath(filepath)
```

Comment:
Please use this function to get only the version. May be create another function to get the actual file to be used.